File: Easy/C/The Rock, Paper, Scissor Game.c
#include <inttypes.h>
#include <stdio.h>

int main() {
    uint8_t n;
    scanf("%" SCNu8, &n);
    for (uint8_t i = 0; i < n; ++i) {
        char Dcoder, You;
        scanf(" %c%c", &Dcoder, &You);
        if (Dcoder == You)
            printf("Draw\n");
        else if ((Dcoder == 'R' && You == 'P') || (Dcoder == 'P' && You == 'S') || (Dcoder == 'S' && You == 'R'))
            printf("You\n");
        else
            printf("Dcoder\n");
    }
}

File: Easy/Rust/The Rock, Paper, Scissor Game.rs
use std::io::{self, BufRead};

fn main() {
    let stdin = io::stdin();
    let mut lines = stdin.lock().lines();
    let n: u8 = lines.next().unwrap().unwrap().trim().parse().unwrap();
    let pairs = lines.next().unwrap().unwrap();
    let mut characters = pairs.chars();
    for _ in 0..n {
        match characters.next().unwrap() {
            'R' => match characters.next().unwrap() {
                'R' => println!("Draw"),
                'P' => println!("You"),
                'S' => println!("Dcoder"),
                _ => unreachable!(),
            },
            'P' => match characters.next().unwrap() {
                'R' => println!("Dcoder"),
                'P' => println!("Draw"),
                'S' => println!("You"),
                _ => unreachable!(),
            },
            'S' => match characters.next().unwrap() {
                'R' => println!("You"),
                'P' => println!("Dcoder"),
                'S' => println!("Draw"),
                _ => unreachable!(),
            },
            _ => unreachable!(),
        }
    }
}
